scopesim/version.py

```diff
@@ -1,2 +1,5 @@
 from importlib import metadata
-version = metadata.version(__package__)
+try:
+    version = metadata.version(__package__)
+except:
+    version = 0.8
```

Comment:
Same as above. We could also just delete the version.py file entirely, as we did in another package recently. But that should then really be a separate PR!

Reply:
Happy to do this